### PR TITLE
Vector:Added one if condition to check if the given speeds are equal …

### PR DESCRIPTION
**Pull request by JameerBabu**
…to zero.

Compiled and ran in my local machine to check errors and removed errors including the changes in dyadic.py and functions.py at the place of
importing.

### CODE REVIEW

File: sympy/physics/vector/dyadic.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 from sympy.core.backend import sympify, Add, ImmutableMatrix as Matrix
 from sympy.core.compatibility import unicode
-from .printing import (VectorLatexPrinter, VectorPrettyPrinter,
+from printing import (VectorLatexPrinter, VectorPrettyPrinter,
```
Comment (asmeurer) [inline sympy/physics/vector/dyadic.py]: This change looks incorrect.

**Comment by asmeurer** [comment]
This looks like the same change as https://github.com/sympy/sympy/pull/14179

**Comment by Abdullahjavednesar** [comment]
I'm closing this as the changes here and #14179 are similar. @JameerBabu you can surely reopen to continue if you do not think so.

**Comment by JameerBabu** [comment]
Can I please know the reason why it is closed ?? Please tell me the things
you want to make so that it can be merged.

**Comment by Abdullahjavednesar** [comment]
I've stated the reason above and even @asmeurer pointed the same. I think you should continue with #14179 to get it merged.

**Pull request closed without merging**